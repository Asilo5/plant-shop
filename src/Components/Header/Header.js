import React from 'react';
import './Header.scss';
import { NavLink } from 'react-router-dom';

const Header = () => {
    return (
        <header>
            <section className='header-section'>
              <h2> Plan(t) your Garden!</h2>
              <NavLink className='link-to-plants' to='/allPlants'>SEE OUR PLANTS</NavLink>
            </section>
         </header>
    )
}

export default Header;