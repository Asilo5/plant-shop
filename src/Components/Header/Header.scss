header {
    background-image: url('https://images.unsplash.com/photo-1503149779833-1de50ebe5f8a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60');
    background-size: cover;
    background-repeat: no-repeat;
    height: 75vh;
    width: 90%;
    margin: 30px auto;
    box-shadow: 5px 5px 5px #888888;
    border: 10px solid white;
    .header-section {
        position: relative;
        top: 15em;
        text-align: center;
    }
    h1 {
      font-size: 6em;
      margin: 0;
      color: white;
      font-family: 'Catamaran', sans-serif;
      margin-bottom: 20px;
    }
    .link-to-plants {
        border: 2px solid white;
        padding: 13px;
        font-size: 2em;
        text-decoration: none;
        color: white;
        &:hover {
            background-color: white;
            color: black;
        }
    }
  }