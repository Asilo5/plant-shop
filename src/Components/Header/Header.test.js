import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {

    it('should match Snapshot', () => {
        let wrapper = shallow(<Header />);

        expect(wrapper).toMatchSnapshot();
    })
})