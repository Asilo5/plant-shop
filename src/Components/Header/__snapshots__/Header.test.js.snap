// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match Snapshot 1`] = `
<header>
  <section
    className="header-section"
  >
    <h2>
       Plan(t) your Garden!
    </h2>
    <NavLink
      className="link-to-plants"
      to="/allplants"
    >
      SEE OUR PLANTS
    </NavLink>
  </section>
</header>
`;
